microsoftgraph/microsoft-graph-explorer-v4
# Fix: Remove lazy loading on feedback button

## Overview
Fixes #2961.
The lazy loading function is affecting the pop up trigger.

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Check out branch
* Run GE on browser (could even be incognito)
* Spend a minute on GE
* Switch to a different tab
* Go back to GE and observe pop up is triggered
